trin-core/src/utils/content_key.rs

```diff
@@ -39,10 +44,20 @@ mod test {
     use super::*;
 
     #[test]
-    fn test_keccak() {
-        let test_input = "302938402a9f";
-        let result = keccak(test_input).unwrap();
-        let inter = hex::decode("3fea4dacec93ff6fd6a7462d1a2cc656f7a13361bb36abf0dde7c27a3ca7606d")
+    fn test_keccak256() {
+        let test_input = hex::decode("302938402a9f").unwrap();
+        let result = keccak256(&test_input);
+        let bytes = hex::decode("924fd25eb4fdce477517e48f6e4d25d2fe53139c7ebcd7a1ac102577a465e4ca")
+            .unwrap();
+        let actual = U256::from(vec_to_array(bytes).unwrap());
+        assert_eq!(actual, result);
```

Comment:
While you didn't change it in this pr, I'm just slightly confused by these names. Typically I prefer using `expected` and `actual`.... between `actual` and `result` it's hard for me to tell which value we're testing against. If you don't mind updating those var names in these two tests